Revamped get started guide to be more similar to typical libraries (#953)
firebase/genkit: docs/get-started.md

# Get started

This guide shows you how to get started with Genkit in a Node.js app.

## Prerequisites

This guide assumes that you're familiar with building applications with Node.js.

To complete this quickstart, make sure that your development environment meets the following requirements:

> * Node.js v18+  

Reviewer: Node.js 20 or later.

Reviewer: ```suggestion
* Node.js v20+  
```

* npm

## Install Genkit dependencies

Install the following Genkit dependencies to use Genkit in your project:

* `genkit` provides the Genkit CLI and tooling  
* `@genkit-ai/ai` and `@genkit-ai/core` provide Genkit core capabilities  
* `@genkit-ai/googleai` provide access to the Google AI Gemini models

> ```

Reviewer 2: Add
```suggestion
```posix-terminal
```

> npm install genkit @genkit-ai/ai @genkit-ai/core @genkit-ai/googleai

Reviewer: installing `npm install genkit` for CLI will need to make sure to say `npx genkit ...` for cli commands.

```

## Configure your model API key

> For this tutorial, we’ll use the Gemini API which provides a generous free tier and does not require a credit card to get started. To use the Gemini API, you'll need an API key. If you don't already have one, create a key in Google AI Studio.

Author: ```suggestion
For this guide, we’ll show you how to use the Gemini API which provides a generous free tier and does not require a credit card to get started. To use the Gemini API, you'll need an API key. If you don't already have one, create a key in Google AI Studio.
```

```

Note: While this tutorial uses the Gemini API from AI Studio, Genkit supports a wide variety of model providers including [Gemini from Vertex AI](https://firebase.google.com/docs/genkit/plugins/vertex-ai#generative_ai_models), Anthropic’s Claude 3 models and Llama 3.1 through the [Vertex AI Model Garden](https://firebase.google.com/docs/genkit/plugins/vertex-ai#anthropic_claude_3_on_vertex_ai_model_garden), open source models through [Ollama](https://firebase.google.com/docs/genkit/plugins/ollama), and several other [community-supported providers](https://firebase.google.com/docs/genkit/models#models-supported) like OpenAI and Cohere.

## Import the library

Import the Genkit core libraries and the plugin for the Google AI Gemini APIs.

```javascript
import { generate } from '@genkit-ai/ai';
import { configureGenkit } from '@genkit-ai/core';
import { googleAI, gemini15Flash } from '@genkit-ai/googleai';
```

## Make your first request

Use the `generate` method to generate a text response.

```javascript
// Make sure to include these imports:
// import { generate } from '@genkit-ai/ai';
// import { configureGenkit } from '@genkit-ai/core';

> // import { googleAI, gemini15Flash } from '@genkit-ai/googleai';

Reviewer 3: nit: I'd probably just leave these uncommented, it's easier for the dev to simply not copy them if they already have them, vs. copying then having to uncomment each one.
